README.md

```diff
@@ -94,6 +97,7 @@ engine = sqlalchemy.create_engine(
     creator=getconn,
 )
 ```
+**Note**: For more examples of using SQLAlchemy to manage connection pooling with the connector, please see [Cloud SQL SQLAlchemy Samples](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/cloud-sql).
```

Comment:
Let's use this page instead https://cloud.google.com/sql/docs/mysql/manage-connections#python

Reply:
How about this page as it shows the postgres version of the code sample? The link you recommended and the one I originally had don't show SQLAlchemy with the connector.
https://cloud.google.com/sql/docs/postgres/connect-connectors#python_1